package com.revature.project4.util;

public class DataSourceException extends RuntimeException{
    public DataSourceException(String message, Throwable cause) {
        super(message, cause);
    }
}
